react-main/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { Field, reduxForm } from 'redux-form';
+
+const required = value => (value ? undefined : 'Field required.');
```

Comment:
Extract these validation functions to a validations file. They will be reused in the whole app, since they are generic. You can place it in rootDir/utils/validations.js